starfish/compat.py

```diff
@@ -98,3 +99,292 @@ def match_histograms(image, reference, multichannel=False):
             matched = _match_cumulative_cdf(image, reference)
 
         return matched
+
+if version.parse(skimage.__version__) > version.parse("0.14.2"):
```

Comment:
Can we not enforce that users have skiimage version > 0.14.2?

Reply:
0.14.2 is the current release, when they next release we'll be able to remove this.